docs/precommit.md

```diff
@@ -7,28 +7,15 @@ You can use Prettier with a pre-commit tool. This can re-format your files that
 
 ## Option 1. [lint-staged](https://github.com/okonet/lint-staged)
 
-**Use Case:** Useful for when you need to use other tools on top of Prettier (e.g. ESLint)
+**Use Case:** Useful for when you want to use other code quality tools along with Prettier (e.g. ESLint, Stylelint, etc.). Supports partially staged files (`git add --patch`)
```

Comment:
If someone needs the partially staged files, they will read the docs.. maybe it's too much info for this page.

Reply:
According to lint-staged feedback, this was one of the most requested features. That's why I wanted to make it more prominent since apparently people are looking for this. I've updated the sentence a bit to make it clear, though. WYT?